docs/en/observability/apm/how-to.asciidoc

```diff
@@ -1,17 +1,47 @@
 [[apm-how-to-guides]]
-== How-to guides
+== APM UI guides
```

Comment:
A quick search in this repo's `docs/en/observability` directory shows 32 uses of `APM UI` and  111 uses of `APM app` (including `{apm-app}`).

@bmorelli25 should we change all the new section titles to use _APM Kibana_ or _APM app_ instead of _APM UI_ (for example, _Navigate the APM app_, _APM app guides_, and _APM app APIs_)?  For some reason I thought _APM UI_ (now _Applications UI_) was more of a serverless thing, but I could be misremembering.

Reply:
APM app is an outdated term so we shouldn't go with that.

Reply:
APM Kibana, perhaps?

Reply:
>APM app is an outdated term so we shouldn't go with that.
Should I find-and-replace the 100+ "APM app" references that are in the docs to avoid confusion? (Either here or in a follow-up PR.)

Reply:
I'd prefer to avoid "APM Kibana" because it will be an extra cleanup step when we port to serverless where the concept of Kibana doesn't exist. We are using "APM UI" in the serverless docs currently. I don't have a strong preference really, but would prefer to avoid diverging serverless from stateful when possible.